frontend/Dockerfile

```diff
@@ -38,7 +38,7 @@ ENV SKIP_PRE_COMMIT=true
 # Get rid of the lockfile as it won't be accurate for the build without workspace.
 # Then install dependencies, and in the process:
 #   - fix the missing lockfile by writing a new one
-RUN pnpm install
+RUN pnpm install && pnpm i18n
```

Comment:
I tried this but it would require more changes.

Since we set the NODE_ENV to production, the `devDependencies` are not installed. This means that `pnpm build` fails because of missing dependencies. Even if we want to move some dependencies from `devDependencies` to `dependencies`, we should do this in Nuxt 3 branch.